### DISCUSSION TITLE
Use logical properties to support rtl

### DISCUSSION THREAD

**Discussion by meyQuery** [tailwindlabs/tailwindcss#6332] [General]
I'm wondering why you didn't use logical properties to solve the right-to-left support problem?

Instead of `ltr:ml-3 rtl:mr-3` we could write `ms-3` to use `margin-inline-start` property.

**Comment by cdes** [comment]
I've been using this plugin for a while now and it's really great.

https://github.com/20lives/tailwindcss-rtl

I think tailwind should use the same mechanism for bidirectional utility classes.

**Reply by ammardev** [reply]
I agree. 

I use this plugin on all of my Tailwind projects. It's very easy and the classes will be cleaner and faster than using `rtl:` or `ltr:` on all horizontal direction-related classes.

**Reply by meyQuery** [reply]
I've used this plugin a few times but I want to see this approach in the official code.
It's frustrating to write directional properties all the time.

**Comment by thibaudcolas** [comment]
This has been raised in the past here #1483 if you want to continue that discussion.

**Reply by meyQuery** [reply]
Thanks. Didn't see that.
